bootstrap/cmd/kfctl/cmd/init.go

```diff
@@ -48,7 +49,20 @@ or a <name>. If just <name> a directory <name> will be created in the current di
 		repo := initCfg.GetString(string(kftypes.REPO))
 		project := initCfg.GetString(string(kftypes.PROJECT))
 		init_gcp := initCfg.GetBool(string(kftypes.SKIP_INIT_GCP_PROJECT))
-		basic_auth := initCfg.GetBool(string(kftypes.USE_BASIC_AUTH))
+
+		useBasicAuth := initCfg.GetBool(string(kftypes.USE_BASIC_AUTH))
```

Comment:
I think these should not be at init, but at generate.
https://github.com/kubeflow/kubeflow/issues/2691#issuecomment-472970551

Reply:
Let's make a follow-up PR for it.  Want to submit it and test with another PR.